### ISSUE TITLE
show tags as a selectable column in list of tickets

### ISSUE THREAD

**Issue by codyliou** [efigence/redmine_issue_tags#1]
I am not sure whether this should work or not (i.e. whether this is a bugreport or a feature request), but when querying the tickets (at an url like https://redmine.example.com/projects//issues), 'tags' does not appear in the list of selectable columns, so as far as I could see, there is no way to display tags in this list (apart from the wordcloud on the sidebar)

**Comment by jacek213** [comment]
Yep, that's a feature request as there was no intention of listing tags as a column in the table :)
